php/libraries/NDB_Page.class.inc

```diff
@@ -514,6 +554,26 @@ class NDB_Page
     }
 
     /**
+     * Creates an HTML radio button but does not add it to the form.
+     *
+     * @param string $field   The fieldname for this radio button
+     * @param string $label   The label to attach to this radio button
+     * @param string $options Options to pass to LorisForm
+     *
+     * @return QuickForm_Element representing radio button
+     */
+    function createRadio($field, $label, $options=null)
```

Comment:
we have wrappers like this for every type. They're required in order to add a layer of abstraction over $this->form so that people don't directly use the LorisForm (and it can eventually be made private..)